components/Onboarding.js

```diff
@@ -28,7 +28,7 @@ const EmergencyCopy = styled.p.attrs({
 })`
   grid-column: 1 / -1;
 `
-const Call = styled.a.attrs({ className: 'border-b' })``
+const Call = styled.a.attrs({ className: 'underline' })``
```

Comment:
@maxgerber you had this as `border-b`, any reason why? might just be a personal pref thing, but I thought it looked a bit weird 😄